cmd/vc-rest/startcmd/params.go

```diff
@@ -155,6 +159,20 @@ const (
 	oAuthSecretFlagUsage     = "oauth global secret, any string. Example: secret-for-signing-and-verifying-signatures"
 	oAuthSecretFlagEnvKey    = "VC_OAUTH_SECRET"
 
+	serverIdleTimeoutFlagName  = "server-idle-timeout"
```

Comment:
This task about metrics provider not needed server idle timeout

Reply:
removed